apps/play/src/components/Button.tsx

```
@@ -0,0 +1,31 @@
+    <button
+      class={`base-button ${props.class ? props.class : ''} ${
+        props.conditionalClass
+          ? props.conditionalClass
+              .filter((c) => c.condition)
+              .map((c) => c.className)
+              .join(' ')
+          : ''
+      }`}
+      onclick={props.onclick}
+      title={props.title ? props.title : 'Button'}
+    >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)